angelsindustries/locale/en/logistic.cfg

```diff
@@ -26,7 +26,36 @@ angels-logistic-chest-buffer=Big buffer chest
 
 angels-crawler=Crawler construction vehicle
 
-angels-lamp=Crawler light
+angels-lamp=Crawler lamp
+
+[entity-description]
+cargo-robot=A slow logistic robot that can carry many items at once.
+cargo-robot-2=A slow logistic robot that can carry many items at once.
```

Comment:
maybe mention cargo size 100 and 200?

Reply:
I don't want to repeat tooltip information in the description. My intention with the description is rather to highlight which information in the tooltips is of relevance with these items. Thus, third party mods could change the specific values without breaking the locales. How about
```
cargo-robot=A slow logistic robot that can carry a stack of items.
cargo-robot-2=A slow logistic robot that can carry a large stack of items.
```
![cargo_robot_tooltip](https://user-images.githubusercontent.com/1843557/102393902-fad02c80-3fa6-11eb-9a6e-bc412a2dc686.png)

Reply:
I didn't know that it showed the capacity! And those descriptions look fine then